docs/docker.md

```diff
@@ -0,0 +1,20 @@
+# Taro in Docker
+
+## Requirements
+- Docker
+- git
+- a running lnd node
+
+## Building from Source
+```
+git clone https://github.com/lightninglabs/taro
+cd taro
+docker build -t taro .
+```
+
+## Starting Docker Container
+### Quick Startup
+```
+docker run --name taro -v /PATH/TO_YOUR/LND/:/lnd/:ro -v /PATH/TO:YOUR/TARO/:/root/.taro/ -e NETWORK=regtest -e HOSTNAME=localhost:10003 --net=host taro
```

Comment:
These now need to be updated to use command line flags instead of environment variables.

Reply:
true. fixed.